index.js

```diff
@@ -0,0 +1,31 @@
+import { Queue } from './queue.js';
+import { StrangeQueue } from './strangeQueue.js';
```

Comment:
`StrangeQueue` naming give zero information about this structure, consider to use smthng like `TwoStackQueue`

Reply:
done